File: tasks/3512-minimum-operations-to-make-array-sum-divisible-by-k.ts
// Accepted 855 / 855 testcases passed Sergey Pomortsev submitted at Jan 26, 2026 20:52 Editorial
// Solution Runtime 1 ms Beats 85.14% Analyze Complexity Memory 57.61 MB Beats 88.19%

const minOperations = (nums: number[], k: number): number => nums.reduce((acc, curr) => acc + curr, 0) % k

File: tasks/3512-minimum-operations-to-make-array-sum-divisible-by-k.go
// Accepted 855 / 855 testcases passed Sergey Pomortsev submitted at Jul 02, 2026 10:54
// Solution Runtime 0 ms Beats 100.00% Memory 8.50 MB Beats 49.54%

func minOperations(nums []int, k int) int {
  var sum int

  for _, num := range nums {
    sum += num
  }

  return sum % k
}